pyspark/test/resources/conf/test_spark-bigdl.conf

```diff
@@ -27,16 +27,6 @@
 # https://github.com/intel-analytics/BigDL/wiki/Programming-Guide#engine
 #
 
-spark.executorEnv.DL_ENGINE_TYPE                    mklblas
```

Comment:
MKL and OMP support some apis for relative environment variables. So we can set it after loading mkl library.